generators/app/templates/test/app.spec.ejs

```diff
@@ -2,12 +2,8 @@
 'use strict';
 
 const fs = require('fs');
-const path = require('path');<% if(testing === 'mocha-chai') {%>
-const chaiHttp = require('chai-http');
-const chai = require('chai');<%} if(orm.sequelize) {%>
+const path = require('path');<% if(orm.sequelize) {%>
 const models = require('../app/models');<%}%>
-<% if(testing === 'mocha-chai') {%>
-chai.use(chaiHttp);<%}%>
```

Comment:
This `chai.use` is not necessary in mocha ? 🤔

Reply:
Fixed, is above